lib/fastlane/plugin/notarize/actions/notarize_action.rb

```diff
@@ -149,6 +154,12 @@ def self.available_options
                                        description: 'Apple ID username',
                                        default_value: username,
                                        default_value_dynamic: true),
+          FastlaneCore::ConfigItem.new(key: :use_app_specific_password,
```

Comment:
Instead of adding an option, how about we check if `FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD` is provided and use it directly if it is?

Reply:
Fixed